pages/advanced-algorithms/available-algorithms/migrate.mdx

```diff
@@ -59,17 +61,31 @@ YIELD row
 RETURN count(row);
 ```
 
+In the case you want to migrate specific results from a SQL query, it is enough to modify the 
+first argument of the query module call:
+
+```cypher
+CALL migrate.mysql('SELECT * FROM example_table', {user:'memgraph',
+        password:'password',
+        host:'localhost',
+        database:'demo_db'} )
+YIELD row
+RETURN count(row);
```

Comment:
Does this give the user the number of records inserted into Memgraph? Or just the records seen in the SQL query in MySQL?

Reply:
It doesn't insert, just bring into memory the rows and the user needs to manipulate it. I changed the query not to display count because it didn't make much sense but to just return plain results. We also have a migration module which we can link to on this page in order for users to see how to persist data in Memgraph.